dev_docs/tutorials/debugging.mdx

```diff
@@ -61,3 +61,49 @@ logging:
     - name: elasticsearch.query
       level: debug
 ```
+
+## Debugging Kibana with APM
+
+Kibana is integrated with APM's node and RUM agents. 
+To learn more about how APM works and what it reports, refer to the [documentation](https://www.elastic.co/observability/solution-overview).
```

Comment:
Given the target audience of this document (developers), does it make more sense to link to [APM's user guide](https://www.elastic.co/guide/en/apm/guide/current/index.html) instead here?